contrib/podmanimage/machine/Containerfile

```diff
@@ -0,0 +1,15 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
+
+RUN dnf -y update; \
+rpm --setcaps shadow-utils 2>/dev/null; \
+dnf -y install podman subscription-manager crun crun-wasm crun-krun chrony \
```

Comment:
long term, i dont think this will work as we will need versioned content ... i.e. podman 5.0, podman 5.1 (or even 5.1.1).  I think fedora-coreos-config does this reasonably nicely with YAML that gets read and converted to what you want above.  perhaps there is some variable magic we can do here.